src/OidcClientSettings.ts

```diff
@@ -130,6 +131,11 @@ export interface OidcClientSettings {
      * Only scopes in this list will be passed in the token refresh request.
      */
     refreshTokenAllowedScope?: string | undefined;
+
+    /**
+     * Set additional custom headers to be passed to the client
+     */
+    customHeaders?: Record<string, CustomHeader>;
```

Comment:
lets name this `extraHeaders` as we already have other extra stuff like: `extraQueryParams` and `extraTokenParams`